README.md

```diff
@@ -102,42 +102,52 @@ Even though in some programming languages is common to name variables or object
     }
 
 ##### Reasoning
-If you're calling a **GET** on `/entities` , you're asking for entities, not an object with the entities inside one of its properties. Think about it as a christmas gift. If you ask santa for all the power rangers action figures ( and i'm talking about the 90's ones ) and you only see one box under the tree, you can't know if you're getting just one or all of them wrapped in one gift. _Also,  ALL the power rangers? Couldn't you just ask for ONE toy?_
+If you're calling a **GET** on `/entities` , you're asking for entities, not an object with the entities inside one of its properties. So don't envelope things that should be being retrieved directly on the object.
```

Comment:
...you are asking for a list of entities, not... ....things that should be retrieved directly.